AndroidHomeApplication/app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,315 @@
+            app:layout_constraintStart_toEndOf="@+id/guidelineStart"
+            app:layout_constraintTop_toBottomOf="@+id/star1" />
+
+        <TextView
+            android:id="@+id/textStorylineDescription"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="4dp"
+            android:text="@string/storyline_description"
+            android:textColor="@color/text_description"
+            android:textSize="14sp"
+            app:layout_constraintEnd_toStartOf="@+id/guidelineEnd"
+            app:layout_constraintStart_toEndOf="@+id/guidelineStart"
+            app:layout_constraintTop_toBottomOf="@+id/textStoryline" />
+
+        <TextView
+            android:id="@+id/textCast"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="24dp"
+            android:text="@string/cast"
+            android:textColor="@color/text_title"
+            android:textSize="14sp"
+            android:textStyle="bold"
+            app:layout_constraintEnd_toStartOf="@+id/guidelineEnd"
+            app:layout_constraintStart_toEndOf="@+id/guidelineStart"
+            app:layout_constraintTop_toBottomOf="@+id/textStorylineDescription" />
+
+        <ImageView
+            android:id="@+id/cast1"
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            android:layout_marginTop="7dp"
+            android:src="@drawable/robert_downey_jr"
+            app:layout_constraintDimensionRatio="1:1"
+            app:layout_constraintEnd_toStartOf="@+id/cast2"
+            app:layout_constraintHorizontal_chainStyle="spread"
+            app:layout_constraintHorizontal_weight="1"
+            app:layout_constraintStart_toEndOf="@+id/guidelineStart"
+            app:layout_constraintTop_toBottomOf="@+id/textCast" />
+
+        <TextView
+            android:id="@+id/textCast1"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="6dp"
+            android:text="@string/robert_downey_jr"
+            android:textColor="@color/text_cast_item"
+            android:textSize="12sp"
+            android:textStyle="bold"
+            app:layout_constraintEnd_toEndOf="@+id/cast1"
+            app:layout_constraintStart_toStartOf="@+id/cast1"
+            app:layout_constraintTop_toBottomOf="@+id/cast1" />
+
+        <ImageView
+            android:id="@+id/cast2"
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            android:layout_marginStart="8dp"
+            android:src="@drawable/chris_evans"
+            app:layout_constraintBottom_toBottomOf="@+id/cast1"
+            app:layout_constraintDimensionRatio="1:1"
+            app:layout_constraintEnd_toStartOf="@+id/cast3"
+            app:layout_constraintHorizontal_weight="1"
```

Comment:
remove app:layout_constraintHorizontal_weight="1"
app:layout_constraintBottom_toBottomOf="@+id/cast1"
here and below